index.html

```diff
@@ -66,5 +66,243 @@ <h2 class="introduction-title">I'm a Sofware Developer</h2>
         </ul>
       </nav>
     </section>
+
+    <section class="work-section">
+      <div id="my-work-box">
+        <h3 class="my-works">My Recent Works</h3>
+        <hr />
+      </div>
+      <div id="story-box">
+        <img src="img/lady-athlet.png" alt="Athlet lady is exercising" />
+        <h4 class="multi-story-heading">Multi-Post Stories</h4>
+        <p class="story-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
```

Comment:
- [ ] Your work section is almost ready, great job! Let's make it even better:

|Template|Your design|
|---|---|
| ![Captura de pantalla 2022-04-21 a la(s) 8 44 33 p  m](https://user-images.githubusercontent.com/83717285/164580616-2e3ad4a1-8021-4373-aa8c-4741467587fa.png) | ![Captura de pantalla 2022-04-21 a la(s) 8 44 43 p  m](https://user-images.githubusercontent.com/83717285/164580642-8e5d0b01-ed17-4039-be03-e34acc714569.png) |

- Let's fix the problem with the font `Crete Round` that is not displaying properly in your design for titles.
- It would be better to use the white background color for this section like in the Figma template.